### DISCUSSION TITLE
README Icons

### DISCUSSION THREAD

**Discussion by hello-smile6** [hello-smile6/hello-smile6#3] [General]
@JaydenDev I liked your icons, so I added some for mine.

**Comment by jdev082** [comment]
> @JaydenDev I liked your icons, so I added some for mine.

Ok?

**Reply by hello-smile6** [reply]
I mean, this. <a href="https://github.com/JaydenDev/JaydenDev/blob/main/README.md?plain=1#L1">
```markdown
[![JaydenDev's Github Stats](https://github-readme-stats.vercel.app/api?username=JaydenDev)](https://jtechnologies.xyz) \
```
</a>
